src/lwc/geBatchNumberSettings/geBatchNumberSettings.html

```diff
@@ -0,0 +1,126 @@
+<template>
```

Comment:
I'm thinking about the "ge" prefix here ... I wonder if the scope of this is really more BDI than GE? This config applies to those using batches outside of Gift Entry, right?

Reply:
Yep, do you think bdiBatchNumberSettings is the way to go?

Reply:
I think I like bdiBatchNumberSettings, let's switch to that!

Reply:
👍🏽